en/assembly/vibration_isolation.md

```diff
@@ -0,0 +1,36 @@
+# Vibration Isolation
+
+This topic shows how to determine whether vibration levels are too high, and lists some simple steps to improve vibration performance.
+
+## Overview
+
+Flight Control boards with in-built accelerometers or gyros are sensitive to vibrations.
+High vibration levels can cause sensor clipping and sensor failures, resulting in a range of problems, from reduced flight efficiency/performance (leading to shorter flights and increased vehicle wear-and-tear), through to fly-aways and crashes.
+
+Airframes are designed to damp/reduce the amplitude of specific structural resonances at the autopilot mounting location.
+Users should further isolate the flight controller from airframe (using foam or some other isolation mount) in order to reduce vibration to the level that the autopilot can handle.
+
+
+## Vibration Analysis
+
+[Log Analysis using Flight Review > Vibration](../log/flight_review.md#vibration) explains how to use logs to confirm whether vibration is a probable cause of flight problems.
+
+
+## Basic Vibration Fixes
+
+A few of simple steps that may reduce vibrations are:
+- Use a vibration-isolation method to mount the autopilot.
```

Comment:
I'd keep the original order: reducing vibrations at the source (e.g. props) is better than adding vibration-isolation for the autopilot.